android/app/src/main/res/layout/layout_nav_bottom_numbers.xml

```diff
@@ -2,7 +2,7 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
-  android:layout_height="wrap_content"
+  android:layout_height="65sp"
```

Comment:
Why wrap_content didn't work, but hard-coding a constant works?

Reply:
Actually overlapping problem was not occurring for every arabic characters. That means bottom nav height was changing with the change in characters inside bottom sheet.

Since for buttons it was already hardcoded height so keeping bottom sheet nav height variable makes no sense and this was the reason of overlapping as well because when bottom sheet size was increasing it was getting overlapped.

I hardcoded the bottom sheet to avoid fluctuations of bottom sheet nav's height with the change in text inside it.

Since nav bar height for buttons was hardcoded so i did the same for bottom sheet as well.

Reply:
Hard-coding may not work well if the system font size is very large, right?

What are the ways to implement it without hard-coding?

Reply:
Since buttons above the bottom sheet are hardcoded where as bottom sheet below it have variable height which is the root cause of problem.

We can do two things to avoid this overlapping

1. Move these two buttons to the right side to avoid overlapping.

I checked on google maps they also have similar kind of bottom sheet navigation layout. To avoid overlapping due to less space they keep all the buttons on right side.

![image](https://github.com/organicmaps/organicmaps/assets/120750626/e7724c46-7844-4e01-9796-cd2185adec91)

This also allows users to use those buttons even in expanded mode

2. We can also make height of buttons dynamic dependent on bottom sheet height so that it always keep a defined margin from the bottom sheet view

Reply:
> What are the ways to implement it without hard-coding?

Will it be okay if i just increase the button height little bit and keep the bottom sheet as wrap content?

Reply:
@biodranik sir please review it once.
According to me there are two options available. Either restructure the landscape navigation ui to make it future proof or solve this problem temporarily until any issue comes

My above solution basically solves around 95% of problem but still if someone enlarges the text size fully then everything dont fits in screen.

Please suggest me what should be done.